### PR TITLE
2020 Toyota Highlander (TSS2)

### PR DESCRIPTION
**Pull request by che220**
Added 2020 Toyota Highlander Limited fingerprint, HIGHLANDER_TSS2. I am not sure if it works on other trims. But the code worked well for about a week. The most similar fingerprint is RAV4_TSS2, which differs from HIGHLANDER_TSS2 by about 19%.

Here are two drives:

Day:
https://my.comma.ai/0a302ffddbb3e3d3/1581178748365/1581181699211

Night:
https://my.comma.ai/0a302ffddbb3e3d3/1581124526006/1581128577901

Both drives performed very well, with OP controlling both lateral and long.

### CODE REVIEW

**Comment by geohot** [comment]
Can you add the testing route in the test_car_models stage? @pd0wm what's the current procedure for this?

**Comment by che220** [comment]
error message: failed to download test log 0a302ffddbb3e3d3|2020-02-08--16-19-08
Not sure how to resolve this:

  "0a302ffddbb3e3d3|2020-02-08--16-19-08": {
    'carFingerprint': TOYOTA.HIGHLANDER_TSS2,
    'enableCamera': True,
    'enableDsu': False,
  },

File: selfdrive/car/toyota/interface.py [hunk 1]
```diff
@@ -163,6 +163,16 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
...
       ret.lateralTuning.pid.kf = 0.00006
 
+    elif candidate == CAR.HIGHLANDER_TSS2:
+      stop_and_go = True
+      ret.safetyParam = 73
+      ret.wheelbase = 2.84988  # 112.2 in = 2.84988 m
+      ret.steerRatio = 16.0
+      tire_stiffness_factor = 0.8
+      ret.mass = 4700. * CV.LB_TO_KG + STD_CARGO_KG  # 4260 + 4-5 people
+      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.18], [0.015]]  # community tuning
```
Comment (pd0wm) [inline selfdrive/car/toyota/interface.py]: Is this tuning correct? The values look really low compared to other TSS2 cars.

**Comment by pd0wm** [comment]
> Can you add the testing route in the test_car_models stage? @pd0wm what's the current procedure for this?

Add the route to the list, and we run the `update_ci_routes.py` script.

File: selfdrive/car/toyota/interface.py (hunk 1 above)
Comment (che220) [reply selfdrive/car/toyota/interface.py]: I have tried the values for other TSS2 cars. When I drive the car on curves, it overturns and corrects all the time, while these lower values are much smoother. I did not notice any ping-ponging on straight roads either. sR 16 is good. The wheel can turn probably 30 degrees with ease.

**Comment by pd0wm** [comment]
Can you also update the readme to include the 2020 model year? Then it's good to merge.

**Comment by pd0wm** [comment]
Can you just change the existing highlander entry by extending the model range to 2020 instead of adding a new line?

**Comment by pd0wm** [comment]
My bad, it's different because the ACC is different. Merged!

**Pull request merged**